redis/src/main/scala/zio/redis/Interpreter.scala

```diff
@@ -29,15 +31,26 @@ trait Interpreter {
     def loopback(port: Int = DefaultPort): ZLayer[Logging, RedisError.IOError, RedisExecutor] =
       (ZLayer.identity[Logging] ++ ByteStream.loopback(port)) >>> StreamedExecutor
 
-    val test: ZLayer[Any, Nothing, RedisExecutor] =
+    val test: ZLayer[zio.random.Random, Nothing, RedisExecutor] = {
+      val makePickRandom: URIO[zio.random.Random, Int => USTM[Int]] =
+        for {
+          seed   <- random.nextInt
+          sRandom = new scala.util.Random(seed)
+          ref    <- TRef.make(LazyList.continually((i: Int) => sRandom.nextInt(i))).commit
+        } yield (i: Int) => ref.modify(s => (s.head(i), s.tail))
```

Comment:
@mschuwalow will provide TRandom. However, until then, I propose to remove random generator completely, and use something not random at all, i.e. long circular buffer of elements.

@mschuwalow wdyt?

Reply:
I think it's fine for now. It's already implemented and we can update it once we have a proper TRandom